src/api.v2/helpers/s3/getLastModified.js

```diff
@@ -0,0 +1,28 @@
+const getS3Client = require('./getS3Client');
+const NotFoundError = require('../../../utils/responses/NotFoundError');
+
+const getLastModified = async (params) => {
```

Comment:
Why is this necessary vs just getting the object from s3?

Reply:
because I don't want to get the whole object, the last modified is enough (head operation).

Reply:
Right. Correct if I'm wrong, but based on what I've seen, `lastModified` is used to then get the last modified object from cell sets, right? My question was about trying to understand in which way this is different from just doing `getObject`

Reply:
Ah ok I understand now, I was confused because the function that was calling this is called `getCellSets`.

I would suggest changing the name for `getCellSets` to something like `getCellSetsVersion` or `getCellSetsLastVersionKey`